tests/tests/blocks.rs

```diff
@@ -77,24 +77,25 @@ async fn block() {
 
 #[tokio::test]
 async fn get_genesis_block() {
```

Comment:
I'm not really sure what the SUT is of this test. Can you clarify what we're testing here in the test name? I don't understand the significance of `14` in `block_by_height` for example.